pages/contribute.md

```diff
@@ -0,0 +1,125 @@
+
+### Stay informed
+
+Stay informed about upcoming opportunities such as **Google Summer of Code, Outreachy, and other mentoring programs**. For the latest announcements, deadlines, and project ideas, check out our [blog]({{ site.baseurl }}/blog/) where we share updates on participation and application guidelines.  
+
+We have been involved in several mentoring initiatives:
+
+- **[Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/)** \
+    A global, remote program where students contribute to open-source projects under mentorship, enhancing their programming skills while engaging with the open-source community.
+
+- **[Google Season of Docs](https://developers.google.com/season-of-docs)** \
+    A program that connects technical writers with open-source projects to improve documentation quality and accessibility.
+
+- **[Outreachy](https://www.outreachy.org/)** \
+    A diversity-focused program offering paid remote internships to individuals underrepresented in tech, providing opportunities to contribute to open-source software.
+
+- **[Station1 Frontiers Fellowship](https://www.station1.org/sff)**  \
+    A unique research and internship program designed for students in science and technology, promoting inclusive and socially impactful research.
+
+- **[CompChemURG](https://www.bindingsites.co.uk/home)**  \
+    A mentoring initiative focused on computational chemistry, supporting undergraduates and early-career researchers in gaining expertise in the field.
+
+Many of our [core developers]({{ site.baseurl }}/governance/#mdanalysis-core-developers) started through these programs &mdash; consider joining and becoming part of the MDAnalysis community!
+
+### GSoC
+
+MDAnalysis has participated in Google Summer of Code for several years, mentoring students on a variety of open-source projects. Explore our past projects:
+
+- [GSoC 2025]({% post_url 2025-02-28-gsoc2025 %})
+- [GSoC 2024]({% post_url 2024-02-27-gsoc2024 %})
+- [GSoC 2023]({% post_url 2023-02-22-gsoc2023 %})
+- [GSoC 2022]({% post_url 2022-03-07-gsoc2022 %})
+- [GSoC 2021]({% post_url 2021-03-09-gsoc2021 %})
+- [GSoC 2020]({% post_url 2020-02-22-gsoc2020 %})
+- [GSoC 2019]({% post_url 2019-03-04-gsoc2019 %})
+- [GSoC 2018]({% post_url 2018-02-14-gsoc2018 %})
+- [GSoC 2017]({% post_url 2017-03-02-gsoc2017 %})
+- [GSoC 2016]({% post_url 2016-04-26-gsoc %})
+
+## Other Contributions
+
+If you are interested in contributing in other ways, such as improving the website or blog, you can refer to the [README file][] in the [Website Repository][]. While the website is already maintained by the MDAnalysis team, any fixes and updates are always welcome. 
+
+If you'd like to write a **blog post**, we welcome contributions that document your experience with MDAnalysis—whether it's learning the software, participating in a UGM, or working on a project. Feel free to reach out via the [Community]({{ site.baseurl }}/pages/community/) page for guidance on submitting a post.
```

Comment:
can we promote (bump up) contributing to the blog (desired) and demote (put last) contributing to the website?